apps/dotcms-ui/src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -63,7 +63,7 @@ export class DotHttpErrorManagerService {
         return of(result);
     }
 
-    private callErrorHandler(response: HttpErrorResponse | any): boolean {
+    private callErrorHandler(response: HttpErrorResponse): boolean {
```

Comment:
This probably broke something.

Reply:
It is a `private method` that is called once and the argument we send is of type `HttpErrorResponse`. However, I'll take a look to be sure.

https://github.com/dotCMS/core-web/blob/087e28bd846d3f7d9a90887c66d0fa5daa17f308/apps/dotcms-ui/src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts#L50-L64